--- lm_sensors-2.8.0.orig/kernel/Module.mk	2003-05-31 17:48:28.000000000 +0200
+++ lm_sensors-2.8.0/kernel/Module.mk	2003-07-25 23:13:59.000000000 +0200
@@ -39,7 +39,7 @@
 install-kernel: all-kernel
 #	if [ -n "$(KERNELTARGETS)" ] ; then \
 #	  $(MKDIR) $(DESTDIR)$(MODDIR) ; \
-#	  $(INSTALL) -o root -g root -m 644 $(KERNELTARGETS) $(DESTDIR)$(MODDIR) ;\
+#	  $(INSTALL) -m 644 $(KERNELTARGETS) $(DESTDIR)$(MODDIR) ;\
 #	fi
 	$(RM) $(DESTDIR)$(MODPREF)/misc/sensors.o
 
--- lm_sensors-2.8.2/Makefile.orig	2003-11-29 20:59:31.000000000 +0100
+++ lm_sensors-2.8.2/Makefile	2003-12-13 23:24:48.024643304 +0100
@@ -183,13 +183,11 @@
 # ARCPPFLAGS/ARCFLAGS are used to create archive object files (static libraries).
 # LIBCPPFLAGS/LIBCFLAGS are for shared library objects.
 ALL_CPPFLAGS := -I. -Ikernel/include -I$(I2C_HEADERS)
-ALL_CFLAGS := -Wall
+OPTS = -O2
+ALL_CFLAGS := -Wall $(OPTS)
 
 ifeq ($(DEBUG),1)
 ALL_CPPFLAGS += -DDEBUG
-ALL_CFLAGS += -O -g
-else
-ALL_CFLAGS += -O2
 endif
 
 ifeq ($(WARN),1)
@@ -328,10 +326,10 @@
 
 # .o files are used for modules
 # depend on the kernel config file!
-%.o: %.c $(LINUX)/.config
+%.o: %.c
 	$(CC) $(MODCPPFLAGS) $(MODCFLAGS) -c $< -o $@
 
-%.d: %.c $(LINUX)/.config
+%.d: %.c
 	$(CC) -M -MG $(MODCPPFLAGS) $(MODCFLAGS) $< | \
        	$(SED) -e 's@^\(.*\)\.o:@$*.d $*.o: Makefile '`dirname $*.d`/Module.mk' @' > $@
 
--- lm_sensors-2.8.5.orig/prog/rrd/Makefile	2002-01-05 16:39:17.000000000 +0100
+++ lm_sensors-2.8.5/prog/rrd/Makefile	2003-07-25 23:17:50.000000000 +0200
@@ -22,7 +22,7 @@
 	sed -e "s#%%RRDPATH%%#$(RRDPATH)#g;s#%%APACHDIR%%#$(APACHDIR)#g;s#%%RRDDIR%%#$(RRDDIR)#g;s#%%MACH%%#$(MACH)#g" $< > $@
 
 $(RRDDIR) :
-	install -d -o $(USER) $(RRDDIR)
+	install -d $(RRDDIR)
 
 $(RRDB) : $(RRDDIR)
 	./sens_create_rrd $(RRDB)
@@ -38,7 +38,7 @@
 	$(error error - Web server directory $(APACHE) not present - check APACHE definition in Makefile)
 
 $(APACHDIR)/pix : $(APACHE)
-	install -d -o $(USER) -m 777 $(APACHDIR)/pix
+	install -d -m 777 $(APACHDIR)/pix
 
 install: all $(RRDB) $(SENSDIR) $(CRONTAB) $(APACHDIR)/pix
 	install -m 755 sens_update_rrd $(BINPATH)
