--- lm_sensors-2.8.6/prog/sensors/Module.mk.orig	2004-03-27 11:30:04.000000000 +0100
+++ lm_sensors-2.8.6/prog/sensors/Module.mk	2004-04-18 17:29:14.681584112 +0200
@@ -33,7 +33,7 @@
 # executables.
 INCLUDEFILES += $(PROGSENSORSSOURCES:.c=.rd)
 
-LIBICONV := $(shell if /sbin/ldconfig -p | grep -q libiconv\\.so ; then echo \-liconv; else echo; fi)
+LIBICONV :=
 
 $(PROGSENSORSTARGETS): $(PROGSENSORSSOURCES:.c=.ro) lib/$(LIBSHBASENAME)
 	$(CC) $(EXLDFLAGS) -o $@ $(PROGSENSORSSOURCES:.c=.ro) $(LIBICONV) -Llib -lsensors
