diff -urN lm_sensors-2.6.5.orig/Makefile lm_sensors-2.6.5/Makefile
--- lm_sensors-2.6.5.orig/Makefile	Sun Sep 22 13:29:54 2002
+++ lm_sensors-2.6.5/Makefile	Sun Sep 22 13:30:32 2002
@@ -170,8 +170,9 @@
 # PROGCPPFLAGS/PROGCFLAGS is to create non-kernel object files (which are linked into executables).
 # ARCPPFLAGS/ARCFLAGS are used to create archive object files (static libraries).
 # LIBCPPFLAGS/LIBCFLAGS are for shared library objects.
+OPTS=-O2
 ALL_CPPFLAGS := -I. -Ikernel/include -I$(I2C_HEADERS) -I$(LINUX_HEADERS)
-ALL_CFLAGS := -O2 
+ALL_CFLAGS := $(OPTS)
 
 ifeq ($(DEBUG),1)
 ALL_CPPFLAGS += -DDEBUG
diff -urN lm_sensors-2.6.5.orig/etc/Module.mk lm_sensors-2.6.5/etc/Module.mk
--- lm_sensors-2.6.5.orig/etc/Module.mk	Sun Sep 22 13:29:54 2002
+++ lm_sensors-2.6.5/etc/Module.mk	Sun Sep 22 13:30:32 2002
@@ -29,7 +29,7 @@
 install-etc:
 	$(MKDIR) $(DESTDIR)$(ETCDIR)
 	if [ ! -e $(DESTDIR)$(ETCINSTALL) ] ; then \
-	  $(INSTALL) -o  root -g root -m 644 $(ETCTARGET) $(DESTDIR)$(ETCINSTALL); \
+	  $(INSTALL) -m 644 $(ETCTARGET) $(DESTDIR)$(ETCINSTALL); \
 	fi
 install :: install-etc
 
diff -urN lm_sensors-2.6.5.orig/kernel/Module.mk lm_sensors-2.6.5/kernel/Module.mk
--- lm_sensors-2.6.5.orig/kernel/Module.mk	Sun Sep 22 13:29:54 2002
+++ lm_sensors-2.6.5/kernel/Module.mk	Sun Sep 22 13:30:32 2002
@@ -39,7 +39,7 @@
 install-kernel: all-kernel
 	if [ -n "$(KERNELTARGETS)" ] ; then \
 	  $(MKDIR) $(DESTDIR)$(MODDIR) ; \
-	  $(INSTALL) -o root -g root -m 644 $(KERNELTARGETS) $(DESTDIR)$(MODDIR) ;\
+	  $(INSTALL) -m 644 $(KERNELTARGETS) $(DESTDIR)$(MODDIR) ;\
 	fi
 	$(RM) $(DESTDIR)$(MODDIR)/sensors.o
 
diff -urN lm_sensors-2.6.5.orig/kernel/busses/Module.mk lm_sensors-2.6.5/kernel/busses/Module.mk
--- lm_sensors-2.6.5.orig/kernel/busses/Module.mk	Sun Sep 22 13:29:54 2002
+++ lm_sensors-2.6.5/kernel/busses/Module.mk	Sun Sep 22 13:30:32 2002
@@ -90,13 +90,13 @@
 	if [ -n "$(KERNELBUSSESTARGETS)" ] ; then \
 	  if [ -d "$(DESTDIR)$(MODPREF)/kernel" ] ; then \
 	    $(MKDIR) $(DESTDIR)$(MODPREF)/kernel/drivers/i2c ; \
-	    $(INSTALL) -o root -g root -m 644 $(KERNELBUSSESTARGETS) $(DESTDIR)$(MODPREF)/kernel/drivers/i2c ; \
+	    $(INSTALL) -m 644 $(KERNELBUSSESTARGETS) $(DESTDIR)$(MODPREF)/kernel/drivers/i2c ; \
 	    for i in $(KERNELBUSSESTARGETS) ; do \
 	      $(RM) $(DESTDIR)$(MODPREF)/misc/`basename $$i` ; \
 	    done \
 	  else \
 	    $(MKDIR) $(DESTDIR)$(MODPREF)/misc ; \
-	    $(INSTALL) -o root -g root -m 644 $(KERNELBUSSESTARGETS) $(DESTDIR)$(MODPREF)/misc ; \
+	    $(INSTALL) -m 644 $(KERNELBUSSESTARGETS) $(DESTDIR)$(MODPREF)/misc ; \
 	  fi \
 	fi
 
diff -urN lm_sensors-2.6.5.orig/kernel/chips/Module.mk lm_sensors-2.6.5/kernel/chips/Module.mk
--- lm_sensors-2.6.5.orig/kernel/chips/Module.mk	Sun Sep 22 13:29:54 2002
+++ lm_sensors-2.6.5/kernel/chips/Module.mk	Sun Sep 22 13:30:32 2002
@@ -137,13 +137,13 @@
 	if [ -n "$(KERNELCHIPSTARGETS)" ] ; then \
 	  if [ -d "$(DESTDIR)$(MODPREF)/kernel" ] ; then \
 	    $(MKDIR) $(DESTDIR)$(MODPREF)/kernel/drivers/sensors ; \
-	    $(INSTALL) -o root -g root -m 644 $(KERNELCHIPSTARGETS) $(DESTDIR)$(MODPREF)/kernel/drivers/sensors ; \
+	    $(INSTALL) -m 644 $(KERNELCHIPSTARGETS) $(DESTDIR)$(MODPREF)/kernel/drivers/sensors ; \
 	    for i in $(KERNELCHIPSTARGETS) ; do \
 	      $(RM) $(DESTDIR)$(MODPREF)/misc/`basename $$i` ; \
 	    done \
 	  else \
 	    $(MKDIR) $(DESTDIR)$(MODPREF)/misc ; \
-	    $(INSTALL) -o root -g root -m 644 $(KERNELCHIPSTARGETS) $(DESTDIR)$(MODPREF)/misc ; \
+	    $(INSTALL) -m 644 $(KERNELCHIPSTARGETS) $(DESTDIR)$(MODPREF)/misc ; \
 	  fi \
 	fi
 
diff -urN lm_sensors-2.6.5.orig/kernel/include/Module.mk lm_sensors-2.6.5/kernel/include/Module.mk
--- lm_sensors-2.6.5.orig/kernel/include/Module.mk	Sun Sep 22 13:29:54 2002
+++ lm_sensors-2.6.5/kernel/include/Module.mk	Sun Sep 22 13:30:32 2002
@@ -33,7 +33,7 @@
 install-all-kernel-include:
 	if [ -n "$(KERNELINCLUDEFILES)" ] ; then \
 	  $(MKDIR) $(DESTDIR)$(SYSINCLUDEDIR) ; \
-	  $(INSTALL) -o root -g root -m 644 $(KERNELINCLUDEFILES) $(DESTDIR)$(SYSINCLUDEDIR) ; \
+	  $(INSTALL) -m 644 $(KERNELINCLUDEFILES) $(DESTDIR)$(SYSINCLUDEDIR) ; \
 	fi
 
 install :: install-all-kernel-include
diff -urN lm_sensors-2.6.5.orig/lib/Module.mk lm_sensors-2.6.5/lib/Module.mk
--- lm_sensors-2.6.5.orig/lib/Module.mk	Sun Sep 22 13:29:54 2002
+++ lm_sensors-2.6.5/lib/Module.mk	Sun Sep 22 13:30:32 2002
@@ -87,12 +87,12 @@
 
 install-lib: all-lib
 	$(MKDIR) $(DESTDIR)$(LIBDIR) $(DESTDIR)$(LIBINCLUDEDIR) $(DESTDIR)$(LIBMAN3DIR) $(DESTDIR)$(LIBMAN5DIR)
-	$(INSTALL) -o root -g root -m 644 $(LIBTARGETS) $(DESTDIR)$(LIBDIR)
+	$(INSTALL) -m 644 $(LIBTARGETS) $(DESTDIR)$(LIBDIR)
 	$(LN) $(LIBSHLIBNAME) $(DESTDIR)$(LIBDIR)/$(LIBSHSONAME)
 	$(LN) $(LIBSHSONAME) $(DESTDIR)$(LIBDIR)/$(LIBSHBASENAME)
-	$(INSTALL) -o root -g root -m 644 $(LIBHEADERFILES) $(DESTDIR)$(LIBINCLUDEDIR)
-	$(INSTALL) -o $(MANOWN) -g $(MANGRP) -m 644 $(LIBMAN3FILES) $(DESTDIR)$(LIBMAN3DIR)
-	$(INSTALL) -o $(MANOWN) -g $(MANGRP) -m 644 $(LIBMAN5FILES) $(DESTDIR)$(LIBMAN5DIR)
+	$(INSTALL) -m 644 $(LIBHEADERFILES) $(DESTDIR)$(LIBINCLUDEDIR)
+	$(INSTALL) -m 644 $(LIBMAN3FILES) $(DESTDIR)$(LIBMAN3DIR)
+	$(INSTALL) -m 644 $(LIBMAN5FILES) $(DESTDIR)$(LIBMAN5DIR)
 
 install :: install-lib
 
diff -urN lm_sensors-2.6.5.orig/prog/detect/Module.mk lm_sensors-2.6.5/prog/detect/Module.mk
--- lm_sensors-2.6.5.orig/prog/detect/Module.mk	Sun Sep 22 13:29:54 2002
+++ lm_sensors-2.6.5/prog/detect/Module.mk	Sun Sep 22 13:30:32 2002
@@ -37,7 +37,7 @@
 
 install-prog-detect: all-prog-detect
 	mkdir -p $(DESTDIR)$(SBINDIR)
-	$(INSTALL) -o root -g root -m 755 $(PROGDETECTSBININSTALL) $(DESTDIR)$(SBINDIR)
+	$(INSTALL) -m 755 $(PROGDETECTSBININSTALL) $(DESTDIR)$(SBINDIR)
 install :: install-prog-detect
 
 clean-prog-detect:
diff -urN lm_sensors-2.6.5.orig/prog/dump/Module.mk lm_sensors-2.6.5/prog/dump/Module.mk
--- lm_sensors-2.6.5.orig/prog/dump/Module.mk	Sun Sep 22 13:29:54 2002
+++ lm_sensors-2.6.5/prog/dump/Module.mk	Sun Sep 22 13:49:15 2002
@@ -39,7 +39,7 @@
 
 install-prog-dump: all-prog-dump
 	mkdir -p $(DESTDIR)$(SBINDIR)
-	$(INSTALL) -o root -g root -m 755 $(PROGDUMPBININSTALL) $(DESTDIR)$(SBINDIR)
+	$(INSTALL) -m 755 $(PROGDUMPBININSTALL) $(DESTDIR)$(SBINDIR)
 install :: install-prog-dump
 
 clean-prog-dump:
diff -urN lm_sensors-2.6.5.orig/prog/rrd/Makefile lm_sensors-2.6.5/prog/rrd/Makefile
--- lm_sensors-2.6.5.orig/prog/rrd/Makefile	Sun Sep 22 13:29:54 2002
+++ lm_sensors-2.6.5/prog/rrd/Makefile	Sun Sep 22 13:31:17 2002
@@ -41,10 +41,10 @@
 	install -d -o $(USER) -m 777 $(APACHDIR)/pix
 
 install: all $(RRDB) $(SENSDIR) $(CRONTAB) $(APACHDIR)/pix
-	install -o root -g root -m 755 sens_update_rrd $(BINPATH)
-	install -o root -g root -m 755 sens_week.cgi $(APACHDIR)
-	install -o root -g root -m 755 sens_day.cgi $(APACHDIR)
-	install -o root -g root -m 755 summ_week.cgi $(APACHDIR)
+	install -m 755 sens_update_rrd $(BINPATH)
+	install -m 755 sens_week.cgi $(APACHDIR)
+	install -m 755 sens_day.cgi $(APACHDIR)
+	install -m 755 summ_week.cgi $(APACHDIR)
 #	grep sens_update_rrd $(CRONTAB) > /dev/null 2>&1 || echo '*/5 * * * *	/usr/local/bin/sens_update_rrd' $(RRDB) $(SENSDEV) >> $(CRONTAB)
 	@echo
 	@echo Note!!! You must manually install the following line in the crontab for user $(USER):
diff -urN lm_sensors-2.6.5.orig/prog/sensord/Module.mk lm_sensors-2.6.5/prog/sensord/Module.mk
--- lm_sensors-2.6.5.orig/prog/sensord/Module.mk	Sun Sep 22 13:29:54 2002
+++ lm_sensors-2.6.5/prog/sensord/Module.mk	Sun Sep 22 13:30:32 2002
@@ -45,8 +45,8 @@
 
 install-prog-sensord: all-prog-sensord
 	$(MKDIR) $(DESTDIR)$(SBINDIR) $(DESTDIR)$(PROGSENSORDMAN1DIR)
-	$(INSTALL) -o root -g root -m 755 $(PROGSENSORDTARGETS) $(DESTDIR)$(SBINDIR)
-	$(INSTALL) -o $(MANOWN) -g $(MANGRP) -m 644 $(PROGSENSORDMAN1FILES) $(DESTDIR)$(PROGSENSORDMAN1DIR)
+	$(INSTALL) -m 755 $(PROGSENSORDTARGETS) $(DESTDIR)$(SBINDIR)
+	$(INSTALL) -m 644 $(PROGSENSORDMAN1FILES) $(DESTDIR)$(PROGSENSORDMAN1DIR)
 install :: install-prog-sensord
 
 clean-prog-sensord:
diff -urN lm_sensors-2.6.5.orig/prog/sensors/Module.mk lm_sensors-2.6.5/prog/sensors/Module.mk
--- lm_sensors-2.6.5.orig/prog/sensors/Module.mk	Sun Sep 22 13:29:54 2002
+++ lm_sensors-2.6.5/prog/sensors/Module.mk	Sun Sep 22 13:30:32 2002
@@ -41,8 +41,8 @@
 
 install-prog-sensors: all-prog-sensors
 	mkdir -p $(DESTDIR)$(BINDIR) $(DESTDIR)$(PROGSENSORSMAN1DIR)
-	$(INSTALL) -o root -g root -m 755 $(PROGSENSORSTARGETS) $(DESTDIR)$(BINDIR)
-	$(INSTALL) -o $(MANOWN) -g $(MANGRP) -m 644 $(PROGSENSORSMAN1FILES) $(DESTDIR)$(PROGSENSORSMAN1DIR)
+	$(INSTALL) -m 755 $(PROGSENSORSTARGETS) $(DESTDIR)$(BINDIR)
+	$(INSTALL) -m 644 $(PROGSENSORSMAN1FILES) $(DESTDIR)$(PROGSENSORSMAN1DIR)
 install :: install-prog-sensors
 
 clean-prog-sensors:
