diff -ur lm_sensors-2.8.8.orig/kernel/busses/i2c-ipmb.c lm_sensors-2.8.8.dev/kernel/busses/i2c-ipmb.c
--- lm_sensors-2.8.8.orig/kernel/busses/i2c-ipmb.c	Mon Mar 17 02:39:57 2003
+++ lm_sensors-2.8.8.dev/kernel/busses/i2c-ipmb.c	Fri Dec  3 09:11:34 2004
@@ -87,7 +87,7 @@
 {
 	int err;
 
-	if((err = ipmi_request(i2c_ipmb_user, address, id, msg, 0)))
+	if((err = ipmi_request(i2c_ipmb_user, address, id, msg, NULL, 0)))
 		printk(KERN_INFO "i2c-ipmb.o: ipmi_request error %d\n",
 			err);
 }
diff -ur lm_sensors-2.8.8.orig/kernel/busses/i2c-ipmi.c lm_sensors-2.8.8.dev/kernel/busses/i2c-ipmi.c
--- lm_sensors-2.8.8.orig/kernel/busses/i2c-ipmi.c	Tue Apr 27 03:11:32 2004
+++ lm_sensors-2.8.8.dev/kernel/busses/i2c-ipmi.c	Fri Dec  3 09:11:43 2004
@@ -82,7 +82,7 @@
 
 static void ipmi_i2c_send_message(int id, struct ipmi_msg * msg)
 {
-	ipmi_request(i2c_ipmi_user, &address, (long) id, msg, 0);
+	ipmi_request(i2c_ipmi_user, &address, (long) id, msg, NULL, 0);
 }
 
 /* This is the message send function exported to the client
