--- lm_sensors-2.8.1/prog/sensord/chips.c.orig	2003-07-29 17:59:51.000000000 +0200
+++ lm_sensors-2.8.1/prog/sensord/chips.c	2003-10-14 20:54:50.119945024 +0200
@@ -953,7 +953,9 @@
 
 static const FeatureDescriptor ddcmon_features[] = {
   { fmtID_DDCMON, NULL, DataType_other, 0, 0,
-    { SENSORS_DDCMON_ID, -1 } },
+    { SENSORS_DDCMON_MAN_ID, -1 } },
+  { fmtID_DDCMON, NULL, DataType_other, 0, 0,
+    { SENSORS_DDCMON_PROD_ID, -1 } },
   { fmtInt, NULL, DataType_other, 0, 0,
     { SENSORS_DDCMON_SERIAL, -1 } },
   { fmtIntCrossInt, NULL, DataType_other, 0, 0,
