diff -uNr lm_sensors-2.6.4.orig/prog/dump/isadump.c lm_sensors-2.6.4/prog/dump/isadump.c
--- lm_sensors-2.6.4.orig/prog/dump/isadump.c	Sat Oct 13 18:37:16 2001
+++ lm_sensors-2.6.4/prog/dump/isadump.c	Sun Aug 25 13:32:57 2002
@@ -31,9 +31,10 @@
 
 
 /* To keep glibc2 happy */
-#if defined(__GLIBC__) && __GLIBC__ == 2 && __GLIBC_MINOR__ >= 0
+#if defined(__GLIBC__) && !defined(__powerpc__) && __GLIBC__ == 2 && __GLIBC_MINOR__ >= 0
 #include <sys/io.h>
 #else
+#define __KERNEL__ 1
 #include <asm/io.h>
 #endif
 
