--- lm_sensors-2.9.2/prog/pwm/pwmconfig.orig	2005-09-06 21:38:10.000000000 +0200
+++ lm_sensors-2.9.2/prog/pwm/pwmconfig	2005-10-07 16:34:43.000000000 +0200
@@ -465,7 +465,7 @@
 function SaveConfig {
 	echo
 	echo "Saving configuration to $FCCONFIG..."
-	tmpfile=`tempfile` || { echo "$0: Cannot create temporary file" >&2; exit 1;  }
+	tmpfile=`mktemp /tmp/pwmconfig.XXXXXX` || { echo "$0: Cannot create temporary file" >&2; exit 1;  }
 	trap " [ -f \"$tmpfile\" ] && /bin/rm -f -- \"$tmpfile\"" 0 1 2 3 13 15
 	egrep -v '(INTERVAL|FCTEMPS|FCFANS|MAXTEMP|MINTEMP|MINSTART|MINSTOP)' $FCCONFIG >$tmpfile
 	echo -e "INTERVAL=$INTERVAL\nFCTEMPS=$FCTEMPS\nFCFANS=$FCFANS\nMINTEMP=$MINTEMP\nMAXTEMP=$MAXTEMP\nMINSTART=$MINSTART\nMINSTOP=$MINSTOP" >>$tmpfile
