--- lm_sensors-3.0.1/Makefile.orig	2008-01-19 11:20:45.000000000 +0100
+++ lm_sensors-3.0.1/Makefile	2008-04-28 19:31:14.999736011 +0200
@@ -54,7 +54,7 @@
 # library files (both static and shared) will be installed.
 LIBDIR := $(PREFIX)/lib
 
-EXLDFLAGS := -Wl,-rpath,$(LIBDIR)
+EXLDFLAGS :=
 
 # You should not need to change this. It is the directory into which the
 # executable program files will be installed. BINDIR for programs that are
@@ -130,13 +130,12 @@
 # ARCPPFLAGS/ARCFLAGS are used to create archive object files (static libraries).
 # LIBCPPFLAGS/LIBCFLAGS are for shared library objects.
 ALL_CPPFLAGS := -I.
-ALL_CFLAGS := -Wall
+OPTS = -O2
+ALL_CFLAGS := -Wall $(OPTS)
 
 ifeq ($(DEBUG),1)
 ALL_CPPFLAGS += -DDEBUG
-ALL_CFLAGS += -O -g
-else
-ALL_CFLAGS += -O2
+ALL_CFLAGS += -g
 endif
 
 ifeq ($(WARN),1)
