diff -urN lm_sensors-2.9.0/prog/detect/sensors-detect lm_sensors-2.9.0.new/prog/detect/sensors-detect
--- lm_sensors-2.9.0/prog/detect/sensors-detect	2004-12-10 21:10:09.000000000 +0100
+++ lm_sensors-2.9.0.new/prog/detect/sensors-detect	2005-02-18 16:35:46.797434032 +0100
@@ -41,6 +41,8 @@
 $ENV{PATH} = '/usr/local/sbin:'.$ENV{PATH}
 	unless $ENV{PATH} =~ m,(^|:)/usr/local/sbin/?(:|$),;
 
+$ENV{PATH} = '/usr/sbin:'.$ENV{PATH}
+	unless $ENV{PATH} =~ m,(^|:)/usr/sbin/?(:|$),;
 #########################
 # CONSTANT DECLARATIONS #
 #########################
